roles/bind-crc-apb/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Retrieve {{ _apb_provision_creds.SERVICE_NAME }} service configuration
+  shell: oc get configmaps/{{ crc_mobile_config_name }} -n {{ namespace }} -o jsonpath='{.data.config}'
+  register: svc_json_str_config
+
+- name: Validate CRC JSON config
+  set_fact: svc_json_config={{ svc_json_str_config.stdout | from_json }}
+  register: crc_json_config_valid
+  ignore_errors: True
+
+- fail:
+    msg: |
+      Failed to validate provided JSON configuration
+      CRC with invalid JSON config can not be created
+  when: crc_json_config_valid|failed
+
+- name: Creating CRC {{ CLIENT_ID }} configmap
+  k8s_v1_config_map:
+    name: '{{ CLIENT_ID | lower }}'
+    namespace: '{{ namespace }}'
+    annotations:
+      configType: '{{ crc_mobile_config_type }}'
+    labels:
+      name: '{{ _apb_provision_creds.SERVICE_NAME }}'
+      mobile: '{{ crc_mobile_svc_label }}'
```

Comment:
not sure on this label, it should be mobile:enabled https://github.com/aerogearcatalog/keycloak-apb/pull/62/files#diff-d03fc7e3ea5b47a1f847b645bdf93da8R7

Reply:
crc_mobile_svc_label variable is moved to vars/ fodler and is crc_mobile_svc_label: 'enabled'. Name can be confusing, suggastions?

Reply:
Ahh ok. No thats fine thanks for explaining